### ISSUE TITLE
Please use English in style setting :)

### ISSUE THREAD

**Issue by Li-Mingshuang** [mgmeyers/obsidian-list-callouts#44]
I have no idea what these mean, and they even can't be copied.
Thank you for your work1
![image](https://github.com/mgmeyers/obsidian-list-callouts/assets/71756865/f34e8b75-c3f9-491b-8466-e9a35b579fcd)

**Comment by berot3** [comment]
I disagree. this is a kind of lorem ipsum or latin, so perfectly for demoing functions.

**Comment by Li-Mingshuang** [comment]
> I disagree. this is a kind of lorem ipsum or latin, so perfectly for demoing functions.

Sorry, I just don't understand what each symbol represents in the author's design. But you reminded me that this is just a display of text.

**Comment by danzbichorski** [comment]
You got me curious, so I had to look it up.

The phrase is:
`sed eu nisl rhoncus consectetur mi qis, scelerisque enim`
and translates to:
`but I am a player, my friend, because I am a chocolate.`

[Google translate of the phrase](https://translate.google.com/?sl=auto&tl=en&text=sed%20eu%20nisl%20rhoncus%20consectetur%20mi%20qis%2C%20scelerisque%20enim&op=translate)
